cards/src/main/resources/cards/custom/group3/bug/verdant_dreams/spell_dream_of_a_feast.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Dream of a Feast",
+  "baseManaCost": 9,
+  "type": "SPELL",
+  "heroClass": "AMBER",
+  "rarity": "EPIC",
+  "description": "Summon six random 2 Attack minions.",
```

Comment:
this is way too strong, that stupid treants card would give you 7 2/2s but also takes away your mana!

Reply:
Makes sense, we're going to move it to 10 mana, which I think is more reasonable. We can keep an eye on it when it goes live, and if needed, we can cut it down to 5 minions if its still too much.